va/validation-authority.go

```diff
@@ -417,7 +421,7 @@ func parseHTTPConnError(err error) probs.ProblemType {
 
 func (va *ValidationAuthorityImpl) validateDNS01(ctx context.Context, identifier core.AcmeIdentifier, challenge core.Challenge) ([]core.ValidationRecord, *probs.ProblemDetails) {
 	if identifier.Type != core.IdentifierDNS {
-		va.log.Debug(fmt.Sprintf("DNS [%s] Identifier failure", identifier))
+		va.log.Err(fmt.Sprintf("identifier type for dns challenge was not dns: %s", identifier))
```

Comment:
You use inconsistent case in these error messages. For better or worse we've been using initial caps, which you should stick to.